addons/presentation/ui/mts_presentation_dialog.hpp

```diff
@@ -0,0 +1,147 @@
+class RscPicture;
+class RscCombo;
+class RscShortcutButton;
+class RscText;
+class RscListBox;
+class IGUIBack;
+
+class GVAR(dialog) {
+    idd = IDD_MAIN_DISPLAY;
+    movingEnable = 0;
+    OnLoad = QUOTE(((_this select 0) displayctrl IDC_LISTBOX) ctrlAddEventHandler [ARR_2('LBSelChanged', {_this call FUNC(fillPagesCombo)})];);
+
+    class Button: RscShortcutButton {
+        type = 16;
+        class HitZone;
+        class ShortcutPos;
+        class TextPos {
+            left = 0;
+            top = 0.0125;
+            right = 0;
+            bottom = 0;
+        };
+        class Attributes
+        {
+            font = "RobotoCondensed";
+            color = "#E5E5E5";
+            align = "center";
+            shadow = "true";
+        };
+        class AttributesImage;
+
+        colorBackground[] = {1, 1, 1, 1};
+        colorBackgroundFocused[] = {1, 1, 1, 1};
+
+        animTextureDefault = QPATHTOF(data\Button_np.paa);
+        animTextureNormal = QPATHTOF(data\Button_np.paa);
+        animTextureDisabled = QPATHTOF(data\Button_p.paa);
+        animTextureOver = QPATHTOF(data\Button_h.paa);
+        animTextureFocused = QPATHTOF(data\Button_h.paa);
+        animTexturePressed = QPATHTOF(data\Button_p.paa);
+    };
+
+    class Listbox: RscListBox {
+        class ComboScrollBar;
+
+        colorBackground[] = {0.231,0.231,0.231,1};
+        colorPictureRight[] = {0, 0, 0, 1};
+        colorTextRight[] = {0, 0, 0, 1};
+        arrowEmpty = "\A3\ui_f\data\GUI\Cfg\Scrollbar\arrowEmpty_ca.paa";
+    	arrowFull = "\A3\ui_f\data\GUI\Cfg\Scrollbar\arrowFull_ca.paa";
+    	thumb = "\A3\ui_f\data\GUI\Cfg\Scrollbar\thumb_ca.paa";
+    	border = "\A3\ui_f\data\GUI\Cfg\Scrollbar\border_ca.paa";
+    	color[] = {1,1,1,0.6};
```

Comment:
Indentation

Reply:
Again, not on view file from Github and in my editor.